vars/Debian.yml

```diff
@@ -1,4 +1,5 @@
 ---
+nginx_package: nginx
```

Comment:
Having the variable in here wouldn't necessarily allow it to be overridden in a playbook/inventory. What's the use case? Is `nginx-extras` a different Nginx package entirely?